modules/library/metadata/src/main/java/org/geotools/util/logging/Log4J2Logger.java

```diff
@@ -140,20 +143,20 @@ private static Level toJavaLevel(final org.apache.logging.log4j.Level level) {
         if (n == StandardLevel.OFF.intLevel()) {
             return Level.OFF;
         } else if (n <= StandardLevel.FATAL.intLevel()) {
-            return Level.SEVERE;
+            return Logging.FATAL;
         } else if (n <= StandardLevel.ERROR.intLevel()) {
             return Level.SEVERE;
         } else if (n <= StandardLevel.WARN.intLevel()) {
             return Level.WARNING;
         } else if (n <= StandardLevel.INFO.intLevel()) {
             return Level.INFO;
-        } else if (n < StandardLevel.DEBUG.intLevel()) {
+        } else if (n <= CONFIG.intLevel()) {
```

Comment:
I don't know where the need for setting up ``operation`` and ``fatal`` came from, but this seems to be the change that would make config messages appear on the DEFAULT_LOGGING profile in GeoServer?

Reply:
I adjusted default logging profile to show config messages in geoserver (it now says level "config" rather than level "info").

Reply:
I skipped over mapping FATAL previously as I did not think we had any custom java levels, but then I noticed OPERATION go by during debugging.